server/src/test/java/org/opensearch/search/SearchServiceTests.java

```diff
@@ -226,7 +226,7 @@ public void onQueryPhase(SearchContext context, long tookInNanos) {
     }
 
     @Override
-    protected Settings nodeSettings() {
+    protected Settings featureFlagSettings() {
```

Comment:
This class needs both, feature flags and settings:

```suggestion
    @Override
    protected Settings featureFlagSettings() {
        return Settings.builder().put(FeatureFlags.CONCURRENT_SEGMENT_SEARCH, true).build();
    }
    
    @Override
    protected Settings nodeSettings() {
         return Settings.builder().put("search.default_search_timeout", "5s").build();
    }
```